utils.js

```diff
@@ -53,6 +53,8 @@ function setPostFields(post, slug) {
   post.attributes.authorAltText = author
     ? author.username + " image"
     : "author";
+  post.attributes.authorBio = author?.bio || "Author bio..";
```

Comment:
Let's leave it blank if the bio is not there. And see how UI will look like if bio is blank

Reply:
it look like 
![image](https://github.com/canopas/canopas-blog/assets/115449373/5c0d0530-ec97-4a91-98be-9f5ca35fb6e9)

Reply:
You can remove  space between them in this case